src/NonlinearSolve.jl

```diff
@@ -37,6 +37,7 @@ include("utils.jl")
 include("raphson.jl")
 include("trustRegion.jl")
 include("levenberg.jl")
+include("pseudoTransient.jl")
```

Comment:
please follow standard naming conventions. `psudotransient.jl` for the page should be fine.